internal/params/config.go

```diff
@@ -583,6 +594,11 @@ func (c *ChainConfig) IsPreStaking(epoch *big.Int) bool {
 	return isForked(c.PreStakingEpoch, epoch)
 }
 
+// IsCrossChain determines whether it is the epoch to support MMR root in the block header
+func (c *ChainConfig) IsCrossChain(epoch *big.Int) bool {
```

Comment:
Change to IsMMR

Reply:
also rename `CrossChainEpoch` to `MMRHeaderEpoch`.

Reply:
> also rename `CrossChainEpoch` to `MMRHeaderEpoch`.

recommend changing the naming to `MMRHardForkEpoch`